From a3403a6cdf7a78085208529bfbd7e8953799b08e Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hmehrtens@maxlinear.com>
Date: Tue, 8 Dec 2020 16:14:20 +0100
Subject: [PATCH] PONRTSYS-8285: Rename mcast_bridge_filter to bridge_filter

This folder should not only be used for multicast bridge filters, but
also other protocols.
---
 drivers/net/datapath/Kconfig                                            | 2 +-
 drivers/net/datapath/{mcast_bridge_filter => bridge_filter}/Kconfig     | 0
 drivers/net/datapath/{mcast_bridge_filter => bridge_filter}/Makefile    | 0
 .../{mcast_bridge_filter => bridge_filter}/mcast_bridge_filter.c        | 0
 drivers/net/ethernet/lantiq/Makefile                                    | 2 +-
 5 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/datapath/Kconfig b/drivers/net/datapath/Kconfig
index 0bef26607a98..71708ceec383 100644
--- a/drivers/net/datapath/Kconfig
+++ b/drivers/net/datapath/Kconfig
@@ -14,5 +14,5 @@ source "drivers/net/datapath/umt/Kconfig"
 source "drivers/net/datapath/mib_counters/Kconfig"
 source "drivers/net/datapath/pon_qos/Kconfig"
 source "drivers/net/datapath/dp_eth_reinsert/Kconfig"
-source "drivers/net/datapath/mcast_bridge_filter/Kconfig"
+source "drivers/net/datapath/bridge_filter/Kconfig"
 endif # INTEL_NP_NET_CORE
diff --git a/drivers/net/datapath/mcast_bridge_filter/Kconfig b/drivers/net/datapath/bridge_filter/Kconfig
similarity index 100%
rename from drivers/net/datapath/mcast_bridge_filter/Kconfig
rename to drivers/net/datapath/bridge_filter/Kconfig
diff --git a/drivers/net/datapath/mcast_bridge_filter/Makefile b/drivers/net/datapath/bridge_filter/Makefile
similarity index 100%
rename from drivers/net/datapath/mcast_bridge_filter/Makefile
rename to drivers/net/datapath/bridge_filter/Makefile
diff --git a/drivers/net/datapath/mcast_bridge_filter/mcast_bridge_filter.c b/drivers/net/datapath/bridge_filter/mcast_bridge_filter.c
similarity index 100%
rename from drivers/net/datapath/mcast_bridge_filter/mcast_bridge_filter.c
rename to drivers/net/datapath/bridge_filter/mcast_bridge_filter.c
diff --git a/drivers/net/ethernet/lantiq/Makefile b/drivers/net/ethernet/lantiq/Makefile
index 7a744884479d..c89d8ee0e079 100644
--- a/drivers/net/ethernet/lantiq/Makefile
+++ b/drivers/net/ethernet/lantiq/Makefile
@@ -21,4 +21,4 @@ obj-$(CONFIG_LTQ_TOE_DRIVER) += ltq_toe_drv.o
 obj-$(CONFIG_INTEL_MIB_COUNTERS) += ../../datapath/mib_counters/
 obj-$(CONFIG_PON_QOS) += ../../datapath/pon_qos/
 obj-$(CONFIG_DP_ETH_REINSERT) += ../../datapath/dp_eth_reinsert/
-obj-$(CONFIG_DP_MCAST_BRIDGE_FILTER) += ../../datapath/mcast_bridge_filter/
+obj-$(CONFIG_DP_MCAST_BRIDGE_FILTER) += ../../datapath/bridge_filter/
